CHARTER.md

```diff
@@ -0,0 +1,192 @@
+TSC votes.
+
+## Section 4. Roles & Responsibilities of the TSC
+
+Subject to such policies as may be set by the CPC, the TSC voting members are responsible for all technical development 
+within the webpack project, including:
+
+* Setting release dates.
+* Release quality standards.
+* Technical direction.
+* Project governance and process.
+* GitHub repository hosting.
+* Conduct guidelines.
+* Maintaining the list of additional Collaborators.
+* Development process and any coding standards.
+* Mediating technical conflicts between Collaborators or Foundation projects.
+
+The TSC voting members will define webpack project's release vehicles.
+
+### Section 4.1. webpack Project Operations
+
+The TSC voting members will establish and maintain a development process for the webpack project. The development 
+process will establish guidelines for how the developers and community will operate. It will, for example, establish 
+appropriate timelines for TSC review (e.g. agenda items must be published at least a certain number of hours in advance 
+of a TSC meeting).
+
+The TSC and entire technical community will follow any processes as may be specified by the OpenJS Foundation Board 
+relating to the intake and license compliance review of contributions, including the OpenJS Foundation IP Policy.
+
+### Section 4.2. Decision-making, Voting, and/or Elections
+
+#### Section 4.2.1. Elections
+
+Leadership roles in the webpack project will be peer elected representatives of the community.
+
+For election of persons (such as the TSC Chairperson), a multiple-candidate method should be used, such as:
+
+* [Condorcet][] or
+* [Single Transferable Vote][]
+
+Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one 
+position to be filled. No election is required if there is only one candidate and no objections to the candidate's 
+election. Elections shall be done within the projects by the Collaborators active in the project.
+
+The TSC voting members will elect from amongst voting TSC members a TSC Chairperson to work on building an agenda for 
+TSC meetings and a OpenJS Cross Project Council (CPC) voting member to represent the TSC in the OpenJS Foundation for a 
+term of one year. The Chair and voting CPC member may be (but are not required to be) the same person. The TSC shall 
+hold annual elections to select a TSC Chairperson and voting CPC member; there are no limits on the number of terms a 
+TSC Chairperson or voting CPC member may serve.
+
+#### Section 4.2.2. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The TSC voting members shall establish 
+appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the 
+development process.
+
+The TSC voting members follow a [Consensus Seeking][] decision making model. When an agenda item has appeared to reach a 
```

Comment:
is "lazy consensus" and "Consensus Seeking decision making model" the same or different?

If it's the same, use the same words. If it is different define both and explain how they differ.

Reply:
fixed.